### PR TITLE
changed the keys in state map for answer pre init

### PR DESCRIPTION
**Pull request by ooanishoo**

### CODE REVIEW

File: .factorypath [hunk 1]
```diff
@@ -0,0 +1,282 @@
+<factorypath>
```
Comment (DevJS) [inline .factorypath:1]: don't commit .factorypath. to be added to giignore

File: src/main/resources/rules/rulesCurrent/shared/10_PreInit/0_PreInit_ANSWER.drl [hunk 2]
```diff
@@ -17,25 +16,25 @@ rule "Pre Init Answer"
...
+		 	String answerValue = firstAnswer.getValue();
 		 	
-		 	if(attributeCode != null && sourceCode != null && targetCode != null && value != null ){
-		 	    rules.set("attributeCode", attributeCode);
-		 	    rules.set("sourceCode", sourceCode);
-		 		rules.set("targetCode", targetCode);
-		 	    rules.set("value", value);
+		 	if(answerSourceCode != null && answerTargetCode != null && answerAttributeCode != null && answerValue != null ){
+		 	    rules.set("answerSourceCode", answerSourceCode);
+		 		rules.set("answerTargetCode", answerTargetCode);
```
Comment (DevJS) [inline src/main/resources/rules/rulesCurrent/shared/10_PreInit/0_PreInit_ANSWER.drl:26]: what about all the rules using the old keys? in ch40 / internmatch / fourdegrees etc. have they been changed?

Comment (ooanishoo) [reply src/main/resources/rules/rulesCurrent/shared/10_PreInit/0_PreInit_ANSWER.drl:26]: ch40 is listening in one rule, Handle Load Selection.

Comment (DevJS) [reply src/main/resources/rules/rulesCurrent/shared/10_PreInit/0_PreInit_ANSWER.drl:26]: ok cool. are you able to change it so i can merge it all ?

**Review by DevJS — APPROVED** [review]
ok

**Pull request merged**